src/service/viewport-impl.js

```diff
@@ -652,8 +671,11 @@ export class ViewportBindingDef {
   /**
    * Updates binding with the new padding.
    * @param {number} unusedPaddingTop
+   * @param {boolean|undefined} unusedOptUpdateScrollPos
```

Comment:
The exact formula between `newPadding`, `oldPadding` and `scrollTop` is not clear yet. And the formula depends on the binding type, passing a `delta` might not be possible because we have different formula for natural binding and ios binding.
I don't know whether we need to pass the `oldPadding` in `binding.updatePaddingTop` or not. If we can make the binding remember the `oldPadding` or have the binding have access to the viewport's `paddingTop_`, then we don't need to pass the `oldPadding`, we can pass a boolean (whether to adjust scroll position) only.